agent/runtime-attach/src/main/java/com/microsoft/applicationinsights/attach/ApplicationInsights.java

```diff
@@ -52,7 +55,10 @@ public static void attach() {
     System.setProperty(RUNTIME_ATTACHED_ENABLED_PROPERTY, "true");
 
     try {
-      Optional<String> jsonConfig = findJsonConfig();
+      Optional<String> jsonConfig = findJsonConfigFromClasspath();
+      if (!jsonConfig.isPresent()) {
```

Comment:
Let's assume the user adds a JSON configuration file in `src/main/resources` and creates a Spring Boot executable JAR.

The user may want to use a configuration different from the default one in the classpath without creating a new JAR file.

It may be useful to check the configuration from the file system first.